addons/payment/tests/test_payment_provider.py

```diff
@@ -67,3 +68,80 @@ def test_provider_not_compatible_when_payment_above_maximum_amount(self):
             self.company.id, self.partner.id, self.amount, currency_id=self.env.company.currency_id.id,
         )
         self.assertNotIn(self.provider, compatible_providers)
+
+    def test_provider_extra_fees_cancelled_if_deactivated(self):
+        """ Test that extra fees of a provider are null if these fees are deactivated.
+        """
+        self.provider.fees_active = False
+        transaction_fees = self.provider._compute_fees(
+            self.amount,
+            self.currency,
+            self.partner.country_id,
+        )
+        self.assertEqual(transaction_fees, 0)
```

Comment:
Since the fees are by default of 0 when not deactivated, it'd be better to update the values to show that they are deactivated without any doubt =]

Reply:
Well spotted!